integration-tests/known-contracts-ProtoALph.ts

```diff
@@ -1,6 +1,6 @@
-export const knownContractProtoALph = "KT1Mkg3dfZL1yTUdYoFCmZ7CQVSpydcMjTR2";
-export const knownBigMapContractProtoALph = "KT18uDJMsdqxU2ox4dvLAn1XSsXK45a9V4Nd";
-export const knownTzip12BigMapOffChainContractProtoALph = "KT1KyeMPwLC7ExVnMr1wQGzCBKukwpDtZ1KJ";
-export const knownSaplingContractProtoALph = "KT1HiTa8uiGDey2DTodDHijTpcjJd62Yx5Rs";
-export const knownOnChainViewContractAddressProtoALph = "KT1TeHWECXnT1cDgigPk1f12jvYBEPR5TXiF";
+export const knownContractProtoALph = "KT1W8LnB5YMYDnNk1Z9BpMkZnWEEWCvZgP9q";
+export const knownBigMapContractProtoALph = "KT1TJSCWVhSKf1vFYHd1DjaMsWYc3YEknCst";
+export const knownTzip12BigMapOffChainContractProtoALph = "KT1KFU8hN2xJoVufPxyeKHLCf5TwELAEquS2";
+export const knownSaplingContractProtoALph = "KT1HpMhQF8jtvC4gYdTTZ1Hatou4i7aXv32X";
+export const knownOnChainViewContractAddressProtoALph = "KT1FsXv8CXL8ALZEFXgKQ4JFLWmH2WdLR7Vu";
```

Comment:
I think these values are generated each time we run integration tests. If so, is it better to exclude them from commits?

Reply:
only changed if we do `npm run originate-known-contracts-and-run-tests` and unsure if every monday the contracts are automatically updated

Reply:
i think we might not need to commit them weekly though, they should also be created during CI I believe. Which is what Ali might be implying